### ISSUE TITLE
COAB instance / metrics not available for service operators

### ISSUE THREAD

**Issue by gcordierOrange** [orange-cloudfoundry/paas-templates#770]
### Expected behavior

* As a paas-templates operator 
* In order to have information about coab instance service
* I need to have metrics for mysql / redis / mongodb / rabbit into the **main** prometheus / grafana

### Observed behavior
In the main grafana/prometheus, I have metrics for  mysql  / mongoDb metrics, but not rabbitMq / redis
In the coab grafana/prometheus, I have metrics for mysql  / mongoDb and **rabbitMq**, but not for redis (not dashboard, but metrics exist)

To sum up,
* RabbitMQ metrcis are not into the main prometheus / grafana
* redis metrics are not into the main grafana and there is no dashbord redis int o the grafana coab
(not very important if metrics are into the main prometheus)
### Affected releases

* 45.0.3
* earlier versions

**Comment by gberche-orange** [comment]
Discovery of metrics url using OSB dashboard_url requires:
* [ ] https://github.com/orange-cloudfoundry/osb-cmdb/issues/46 for service instance upgrade
* [ ] https://github.com/orange-cloudfoundry/cf-ops-automation-broker/issues/47 supported by https://github.com/orange-cloudfoundry/cf-ops-automation-broker/pull/157
* [ ] model update to 
   * [ ] include a prometheus + grafana instance collocated with broker 
   * [ ] include exporters on the nodes

**Comment by JCL38-ORANGE** [comment]
will be delivered in v47.

**Comment by JCL38-ORANGE** [comment]
merged in v47
